### PR TITLE
feat(service-catalog): Add k8s plugin annotation to peribolos

### PR DESCRIPTION
**Pull request by SamoKopecky**
Part of https://github.com/operate-first/service-catalog/issues/12

SSIA

### CODE REVIEW

File: service-catalog/components/peribolos-as-a-service.yaml [hunk 1]
```diff
@@ -7,6 +7,7 @@ metadata:
     operate-first.cloud/logo-url: https://raw.githubusercontent.com/operate-first/peribolos-as-a-service/main/static/robot.png
     argocd/app-name: peribolos-as-a-service-smaug
     github.com/project-slug: operate-first/peribolos-as-a-service
+    backstage.io/kubernetes-label-selector: 'app.kubernetes.io/name: peribolos-as-service'
```
Comment (tumido) [inline service-catalog/components/peribolos-as-a-service.yaml:10]: This is not a valid label selector. I think it should be:

```suggestion
    backstage.io/kubernetes-label-selector: 'app.kubernetes.io/name=peribolos-as-service'
```

**Review by tumido — CHANGES_REQUESTED** [review]

File: service-catalog/components/peribolos-as-a-service.yaml (hunk 1 above)
Comment (SamoKopecky) [reply service-catalog/components/peribolos-as-a-service.yaml]: Yea thats right, I just copied it without checking. I will pay more attention next time.

**Comment by SamoKopecky** [comment]
Unblocked, ready for review again.

**Review by tumido — APPROVED** [review]
/lgtm
/approve

**Pull request merged**